net/ghttp/ghttp.go

```diff
@@ -101,7 +101,7 @@ type (
 	}
 
 	// ServerStatus is the server status enum type.
-	ServerStatus int
+	ServerStatus = int
```

Comment:
?

Reply:
That means type `ServerStatus` is alias of std type `int`, called `alias type`. The `alias type` cannot be a receiver, which means there cannot be function definitions on it.